Documentation/git-p4.txt

```diff
@@ -638,6 +638,19 @@ git-p4.pathEncoding::
 	to transcode the paths to UTF-8. As an example, Perforce on Windows
 	often uses "cp1252" to encode path names.
 
+git-p4.clDescEncoding::
+	Perforce allows non-unicode characters in changelist description. 
+	Use this config to tell git-p4 what encoding Perforce had used for 
+	description text. This encoding is used to transcode the text to
+	UTF-8. Defaults to "utf_8".
+
+git-p4.clDescNonUnicodeHandling::
```

Comment:
Configuration name of the document not match with the Py file. Should be clDescEncodingErrHandling here?

Reply:
@joelliang would you terribly mind reviewing on the Git mailing list instead? See the ["reply to this" instructions](https://github.com/gitgitgadget/gitgitgadget/wiki/ReplyToThis) and find the mail that you want to reply to [here](https://lore.kernel.org/git/pull.864.git.1612371600332.gitgitgadget@gmail.com/).